### ISSUE TITLE
Code Lab Feedback

### ISSUE THREAD

**Issue by zacharyweston** [zacharyweston/codelab-html-intro#1]
Hey @chrisvfritz, can you take a look at this? It's hosted at http://acid-yak.surge.sh/ and meets the following [project](http://lansingcodelab.com/lessons/html-intro/1) criteria:
- [x] Use at least one `h1` element.
- [x] Use at least one `a` element.
- [x] Use at least one `p` element.
- [x] Use at least one `img` element.
- [x] Use at least one `ul` element.
- [x] Use at least one `li` element.
- [x] When you paste the address of your website into [the W3C validator](https://validator.w3.org/), it returns no errors (warnings are OK).
- [x] This is the last time I'll mention it, but once again, your project should be on GitHub and hosted online (I recommend using surge again).

**Comment by chrisvfritz** [comment]
No errors here, but there are a few nitpicky things we can do to make this project _perfect_. First, I see two spaces after sentences, which isn't actually standard on the Internet. In fact, the web browser will only display one of those spaces, ignoring the other completely. Try removing the double space after the period on [this line](https://github.com/zacharyweston/codelab-html-intro/blob/master/index.html#L11). Also, since [the unordered list](https://github.com/zacharyweston/codelab-html-intro/blob/master/index.html#L13-L17) isn't actually _inside_ of the `h2`, I would dedent (aka unindent) those lines to be on the same level as the `h2`.

Let know if all of that makes sense and let me know when it's updated! :smiley:

**Comment by zacharyweston** [comment]
Makes sense, the space has been removed and the <ul> tags have been un-indented.

**Comment by zacharyweston** [comment]
Hosted at http://last-wave.surge.sh/

**Comment by chrisvfritz** [comment]
:shipit:
